src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -70,65 +70,109 @@ and a guide for using macros can be found at
+| **REF**      | Reference channel                                                                                                               |
+| **OTHER**    | Any other type of channel                                                                                                       |
 
-If there are photos of the electrodes, the [`_acq-<label>`](../appendices/entities.md#acq) entity should be specified with:
 
--   `*_photo.jpg` for an operative photo.
--   `*_acq-xray#_photo.<extension>` for an X-ray picture.
--   `*_acq-drawing#_photo.<extension>` for a drawing or sketch of electrode placements.
--   `*_acq-render#_photo.<extension>` for a rendering.
+## Electrodes  (`*_electrodes.tsv`)
 
-The file `<extension>` for photos MUST be either `.jpg`, `.png`, or `.tif`.
+Electrodes are the physical recording sites that make electrical contact with neural tissue to capture electrophysiological signals.
 
-The [`ses-<label>`](../appendices/entities.md#ses) entity may be used to specify when the photo was taken.
+The electrode positions and properties are stored in a `.tsv` file (amplifier information is in `channels.tsv`).
 
-## **Multiple coordinate systems**
+This file contains the following information:
 
-The optional[` space-<label>`](../appendices/entities.md#space) entity (`*[_space-<label>]_coordsystem.json`) can be used to indicate how to interpret the electrode positions. The space `<label>` MUST be taken from one of the modality specific lists in the [Coordinate Systems Appendix](../appendices/coordinate-systems.md). For example for iEEG data, the restricted keywords listed under [iEEG Specific Coordinate Systems](../appendices/coordinate-systems.md#ieeg-specific-coordinate-systems) are acceptable for `<label>`.
+-   The electrode name
+-   The electrode coordinates in 3 columns (`xyz`) (use `n/a` for values if a dimension is absent). By default, this is the position on the probe (not the brain).
+-   The ID of the probe the electrode is located on
+
+
+{{ MACROS___make_columns_table("microephys.microephysElectrodes") }}
+Example of `*_electrodes.tsv`:
+
+```tsv
+electrode_id	probe_id	impedance	x	y	z	material	location
+e0123	p01	1.1	-11.87	-1.30	-3.37	iridium-oxide	V1
+e234	p01	1.5	-11.64	0.51	-4.20	iridium-oxide	V2
+e934	p02	3.5	-12.11	-3.12	-2.54	iridium-oxide	V4
+e234	p02	7.0	-9.94	-1.19	-2.86	iridium-oxide	V3
+```
 
 ## Probes (`*_probes.tsv`)
 
-Probes are physical devices used for recording microephys data.
-They can be permanently implanted (chronic recordings) or inserted temporarily for the recording (acute recordings).
+Probes are electrode-bearing devices that interface with neural tissue to record electrophysiological activity, ranging from multi-electrode arrays to single recording pipettes. They can be permanently implanted (chronic recordings) or inserted temporarily for the recording (acute recordings).
```

Comment:
note: although I do not think it is adopted wide practice yet,  it is better to stick to one sentence per line -- minimizes patches etc.

Reply:
@poldrack even now created https://pypi.org/project/mdnewline/